### PR TITLE
Inserter: add tabs.

### PR DESCRIPTION
**Pull request by mtias**
Adds recent, blocks, and embeds.

See #1325.

### CODE REVIEW

**Comment by mtias** [comment]
Sounds like we'd want fixed min-height with this change.

**Comment by jasmussen** [comment]
Pushed a little polish. Props @jwold. It's now good looking.

There's an issue, though:

screen shot 2017-06-29 at 13 47 38

All the blocks fade out when you use the in-stream inserter. @mtias can you take a look?

File: editor/inserter/style.scss [hunk 1]
```diff
@@ -137,6 +138,14 @@ input[type="search"].editor-inserter__search {
 	width: 100%;
 
 	&.is-active {
-		border-bottom: 2px solid #f00;
+		background: $white;
+		border-bottom: 2px solid $blue-medium-500;
```
Comment (paulwilde) [inline editor/inserter/style.scss]: Not a big deal, but this is cleaner if it were just border-bottom-color: $blue-medium-500.

**Comment by paulwilde** [comment]
See #1574 regarding the fade out bug. It's unrelated to this pull request.

**Comment by paulwilde** [comment]
I'm seeing a couple of visual issues.

Firstly on Safari 10.1.2 there is some top/bottom margin. Adding `margin: 0;` to `.editor-inserter__tab` fixes the issue.

screen shot 2017-06-29 at 13 27 15

Secondly, the active state turns the text colour to white.

**Comment by paulwilde** [comment]
I personally prefer it visually if the active tab and content bleed together. I have the CSS written that achieves this and fixes the bugs mentioned above if interested.

screen shot 2017-06-29 at 13 35 22

File: editor/inserter/style.scss (hunk 1 above)
Comment (jasmussen) [inline editor/inserter/style.scss]: 👍 👍

**Comment by jasmussen** [comment]
Thanks for the feedback, addressed most of it.

I think we should address the white bleeding in to the tab separately, as we are likely to move the tabbar around a bit: https://github.com/WordPress/gutenberg/issues/1325#issuecomment-311946486

I also relaxed the margins and paddings a bit:

screen shot 2017-06-29 at 15 16 25

**Pull request merged**